src/components/network-modification-tree.js

```diff
@@ -38,6 +40,14 @@ const NetworkModificationTree = (props) => {
         height: '100%',
     };
 
+    const stylePointerGrab = {
+        cursor: 'grab',
+    };
+
+    const stylePointerGrabbing = {
+        cursor: 'grabbing',
+    };
+
```

Comment:
Perhaps it's better to regroup  styles into an object like this :
`    const styles = {
           container: {
            width: '100%',
            height: '100%',
        },
       flow: {
            cursor: isMoving ? 'grabbing' : 'grab',
        },
    };`

Reply:
I agree. Fixed.